gateway-service/src/main/resources/configs/common/application.conf

```diff
@@ -69,6 +69,10 @@ entity.idcolumn.config = [
   },
 ]
 
+filter.entity = {
```

Comment:
entity.andFilter.enabled?

Reply:
I usually follow a top-down hierarchial approach while building configs, so that the configs are re-usable at a parent level

In this case, `filter` being the parent can be extended to other configs, and so on

Reply:
While reading the code, all variable names are isEntityAndFilterEnabled, but the corresponding configuration name differs in the application.conf file. This makes it a bit difficult to easily co-relate.